### ISSUE TITLE
This does not seem to work with the latest ollama

### ISSUE THREAD

**Issue by gerroon** [ollama-ui/ollama-ui#31]
I see that it sends a /api/tags but it does not follow up or generate responses. Also the connection to the server seems a bit random as in sometimes it connects sometimes it does not.

I tested on ollama WSL2, Brave  [Version 1.61.91 Chromium: 119.0.6045.163 (Official Build) (64-bit)](https://brave.com/latest/)

**Comment by oliverbob** [comment]
I totally disagree. I use the latest ollama and it works. Except that there is no improvement for the attachment or multimodal support.

**Comment by gerroon** [comment]
Thanks I will try it again.

**Comment by mrdjohnson** [comment]
So I think this is device specific and based on this implementation. See this [PR](https://github.com/mrdjohnson/llm-x/pull/9/files).

My app was based on the work in this app and I had a user that ran into this issue a lot (especially during image to text). 

For some reason json objects might come in batches! 
so instead of 
```
{...expected response ...}
```
it would try to parse 
```
{...expected response line 1...}
{... expected response line 2...}
```
which of course is not valid json and therefore fails.

This never happens for me but happened a lot of them, I think the solution is to probably just switch to using the ollama js or langchain implementations. (or do what we did and manually handle this case lol) 

I would have investigated more but I could not reproduce :/
